src/components/lectures/LecturesList/index.tsx

```diff
@@ -68,90 +97,93 @@ const LecturesList: React.FC<LecturesListProps> = ({ course, refreshData }) => {
   };
 
   return (
-    <div className="bg-white w-full rounded-lg shadow-md overflow-hidden mx-auto">
-      <div className="py-4 px-6">
-        <div className="mb-8 flex justify-evenly">
-          <button
-            onClick={() => setModalStatus({ open: true, type: 'single' })}
-            className="py-2 px-4 bg-red-500 text-white rounded hover:bg-gray-600 focus:outline-none flex items-center"
-            type="button"
-          >
-            <FaPlus />
-            <span className="mx-1">{t('addLecture')}</span>
-          </button>
-          <button
-            onClick={() => setModalStatus({ open: true, type: 'import' })}
-            className="py-2 px-4 bg-blue-500 text-white rounded hover:bg-gray-600 focus:outline-none flex items-center"
-            type="button"
-          >
-            <FaSyncAlt />
-            <span className="mx-1">{t('importFromYoutube')}</span>
+    <DndProvider backend={HTML5Backend}>
+      <div className="bg-white w-full rounded-lg shadow-md overflow-hidden mx-auto">
+        <div className="py-4 px-6">
+          <div className="mb-8 flex justify-evenly">
+            <button
+              onClick={() => setModalStatus({ open: true, type: 'single' })}
+              className="py-2 px-4 bg-red-500 text-white rounded hover:bg-gray-600 focus:outline-none flex items-center"
+              type="button"
+            >
+              <FaPlus />
+              <span className="mx-1">{t('addLecture')}</span>
+            </button>
+            <button
+              onClick={() => setModalStatus({ open: true, type: 'import' })}
+              className="py-2 px-4 bg-blue-500 text-white rounded hover:bg-gray-600 focus:outline-none flex items-center"
+              type="button"
+            >
+              <FaSyncAlt />
+              <span className="mx-1">{t('importFromYoutube')}</span>
+            </button>
+          </div>
+          <h1 className="text-3xl  text-gray-800 font-extrabold ">
+            {t('lectures')}
+          </h1>
+          <hr />
+          <div>
+            <StyledLectureList>
+              {/* {orderBy(lectures, 'position', 'asc').map((lecture, index) => ( */}
+              {lectures.map((lecture, index) => (
+                <LectureItem
+                  key={lecture.id}
+                  id={lecture.position}
+                  index={index}
+                  lecture={lecture}
+                  handleDelete={() => setLectureForDelete(lecture)}
+                  moveCard={moveCard}
+                  handlePreview={handlePreview}
+                />
+              ))}
+            </StyledLectureList>
+          </div>
+          <button type="submit" onClick={handleClick}>
+            update
           </button>
```

Comment:
- Can we have a bit of style for this button, please. 
- Change the text to `Update order` instead of just update and make sure it's not hard-coded text.